### PR TITLE
Yogesh

### PR DESCRIPTION
**Pull request by anupkore**

### CODE REVIEW

File: src/App.js
Comment (Techvarun1991) [inline src/App.js:1]: Remove unwanted commented part in the App.js fiile

File: src/Components/AllProject.js
Comment (Techvarun1991) [inline src/Components/AllProject.js:1]: Remove unused imports

File: src/Components/LoginForm.js [hunk 1]
```diff
@@ -32,31 +34,42 @@ export const  LoginForm = ({ toggleSignup }) =>
     
     function handleSignIn(event)
```
Comment (Techvarun1991) [inline src/Components/LoginForm.js:35]: Add comments to describe the function

File: src/Components/ProjectCards.js
Comment (Techvarun1991) [inline src/Components/ProjectCards.js:1]: Kindly check the componenet has all commemted part it is non working code if not required please delete it..

**Review by Techvarun1991 — CHANGES_REQUESTED** [review]
Kindly check and correct the following

File: src/App.js
Comment (anupkore) [reply src/App.js:1]: ok. working on it

File: src/Components/AllProject.js
Comment (anupkore) [reply src/Components/AllProject.js:1]: ok

File: src/App.js
Comment (anupkore) [reply src/App.js:1]: Commented code removed

File: src/Components/AllProject.js
Comment (anupkore) [reply src/Components/AllProject.js:1]: Removed

File: src/Components/LoginForm.js (hunk 1 above)
Comment (anupkore) [reply src/Components/LoginForm.js:35]: > Add comments to describe the function

ok

File: src/Components/ProjectCards.js
Comment (anupkore) [reply src/Components/ProjectCards.js:1]: Commented code removed

File: src/Components/LoginForm.js (hunk 1 above)
Comment (anupkore) [reply src/Components/LoginForm.js:38]: Description added

**Review by Techvarun1991 — APPROVED** [review]

**Pull request merged**